org.opentosca.container.util/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Container Utils
+Bundle-SymbolicName: org.opentosca.container.util
+Bundle-Version: 1.0.0.qualifier
+Automatic-Module-Name: org.opentosca.container.util
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Export-Package: org.opentosca.container.util.converter.impl, org.opentosca.container.util.converters, org.opentosca.container.util.formatters, org.opentosca.container.util.impl
```

Comment:
Usually you would prefer to not export the impl packages in favor of defining only service implementations in them and having dependents only be aware of the interface for the service you're providing